Package.swift

```diff
@@ -355,6 +364,18 @@ extension Array where Element == PackageDescription.SwiftSetting {
 
     return result
   }
+
+  /// Settings necessary to enable Strict Memory Safety, introduced in
```

Comment:
Do we need a GitHub issue for follow-up here?

Reply:
I don't think so, it's just a low priority clean up opportunity the next time we bump our minimum toolchain (to adopt the official `.strictMemorySafety()` API).